api/api.py

```diff
@@ -4,10 +4,9 @@
 
 # flask start
 app = Flask(__name__)
-app.config["DEBUG"] = True
+app.config["DEBUG"]=True
 
```

Comment:
No need to delete spaces here. You can install pylint locally to check automatically code formatting before committing your change. Please check this link https://pre-commit.com/ [Super important and best practice]